### ISSUE TITLE
The type 'ThemeResource' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built

### ISSUE THREAD

**Issue by sigmarsson** [unoplatform/uno#8774]
### Current behavior

1. Created a page for tab navigation. 
2. Set ```Page``` ```Background``` as    ```Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"```
3. Encountering a compilation error on `Wasm `as 

![image](https://user-images.githubusercontent.com/43340959/168444449-45dfbb20-3898-4846-9f9b-7cbcdae673dd.png)

### Expected behavior

It ought to work like on Windows Desktop. 

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia, Uno.UI.RemoteControl / Uno.WinUI.RemoteControl

### NuGet package version(s)

```
    <ItemGroup>
      <PackageReference Include="DryIoc.dll" Version="4.8.8" />
      <PackageReference Include="LiveChartsCore.SkiaSharpView.Uno" Version="2.0.0-beta.220" />
      <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
      <PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.0" />
      <PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.3.0" />
      <PackageReference Include="Uno.WinUI.WebAssembly" Version="4.2.6" />
      <PackageReference Include="Uno.WinUI.RemoteControl" Version="4.2.6" Condition="'$(Configuration)'=='Debug'" />
      <PackageReference Include="Uno.Wasm.Bootstrap" Version="3.3.1" />
      <PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="3.3.1" />
    </ItemGroup>
```

### Affected platforms

WebAssembly

### IDE version

Version 17.2.0

**Comment by jeromelaban** [comment]
Note that the repro cannot work, since LiveChartsCore.SkiaSharpView.Uno references Uno.UI, not Uno.WinUI. LiveCharts can't yet be updated until https://github.com/mono/SkiaSharp/issues/1787 is fixed. 

Make sure to validate with Uno.UI first.

**Comment by lotrasoft** [comment]
This happens for me as well. The project compiles. It is an **Intellisense** error only. Looks like there are some issues in Uno libraries that make **Intellisense** not work correctly. Lots of this type of errors happen.

**Comment by jeromelaban** [comment]
@lotrasoft that's correct, it's a Visual Studio issue that we've working with Microsoft to improve. If you select the Windows project in the top left selector in the XAML editor, this error should go away.

**Comment by lotrasoft** [comment]
Thanks, but there is no selector on XAML editor:

![image](https://user-images.githubusercontent.com/2897950/212141894-f0d23e03-9186-4e21-8cf0-bc18150cdeb1.png)

**Comment by jeromelaban** [comment]
This is curious. Where is the file located in your solution? In which project, specifically?

**Comment by lotrasoft** [comment]
@jeromelaban The Xaml file is located in the Shared project. And note that the WYSIWYG editor for XAML is also not available.

**Comment by jeromelaban** [comment]
It is expected that the designer is not available https://github.com/microsoft/microsoft-ui-xaml/issues/5917, but the curious part is the missing top level bar. Can you try closing all the editor files, then close visual studio, then reopen the solution?

**Comment by lotrasoft** [comment]
I have done that 10000 times. Does not help. Note that with Uno, a few internal components of Visual Studio crashes. Here is an example (look at the brown bars):

![image](https://user-images.githubusercontent.com/2897950/212162512-c871a14b-581c-4f16-9077-10d8b4f6c182.png)

**Comment by jeromelaban** [comment]
Thanks for the update. Two things:
- Can you try unloading all projects except the shared project and the windows project and see if it helps?
- Can you attach here the two "Show stack trace" in blue that VS is showing at the top?

**Comment by lotrasoft** [comment]
I just installed **Visual Studio update 17.4.4**, and looks like the problem has been solved. The XAML toolbar now shows up and the **ThemeResource** error is gone.

![image](https://user-images.githubusercontent.com/2897950/212171395-51529cf1-4a10-49b2-b0d4-1147bcd02827.png)

**Comment by muhammet484** [comment]
I have same error.
![image](https://user-images.githubusercontent.com/51961772/214088307-2455dcab-ea0e-4f9d-b9a1-f12c8e396c8b.png)
Mostly same with lotrasoft but my vs is that "Microsoft Visual Studio Community 2022 (64-bit) - Preview
Version 17.5.0 Preview 3.0"
Using libraries are like this:
![image](https://user-images.githubusercontent.com/51961772/214089031-993804a0-fec6-4463-8fd7-d667692bb95c.png)

**Comment by jeromelaban** [comment]
@muhammet484 make sure to select the [`.Windows` project in the top left selector](https://platform.uno/docs/articles/get-started-wizard.html#the-xaml-editor-shows-the-type-page-does-not-support-direct-content-message).

**Comment by muhammet484** [comment]
> @muhammet484 make sure to select the [`.Windows` project in the top left selector](https://platform.uno/docs/articles/get-started-wizard.html#the-xaml-editor-shows-the-type-page-does-not-support-direct-content-message).

do you mean this? I selected the windows already
![image](https://user-images.githubusercontent.com/51961772/214091312-326bee4d-8857-4509-988c-e8f602b376e3.png)
you can also check my screen:
![image](https://user-images.githubusercontent.com/51961772/214091403-e3c05d54-8f6f-4589-8d29-809a3756f8c9.png)

**Comment by jeromelaban** [comment]
@muhammet484 this selector, specifically:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/5839577/214093025-a1ecedbd-fd4d-4831-aab6-76ed213776c4.png">

**Comment by muhammet484** [comment]
thanks it solved the problem.

**Comment by dallyh** [comment]
> @muhammet484 this selector, specifically: <img alt="image" width="339" src="https://user-images.githubusercontent.com/5839577/214093025-a1ecedbd-fd4d-4831-aab6-76ed213776c4.png">

I would like to address this issue a little more. Let's say I'd like to build apps only for mobile platforms (Android/iOS) because I'm not interested in developing an app for other platforms. When I generate a new project from template using custom settings (just Android and iOS as platforms), then this workaround doesn't work, because i can select "Windows" nor "UWP" from the projects dropdown.  Is there anything known I can do with this?

![image](https://github.com/unoplatform/uno/assets/6968534/2eb8fd09-5a43-45af-8e2b-57e0a73274eb)
![image](https://github.com/unoplatform/uno/assets/6968534/bb3872f5-c2d3-4a83-9db1-d7ed0da71825)

**Comment by MartinZikmund** [comment]
@dallyh This is unfortunately true, we are using built-in XAML intellisense provided by Microsoft, which is only compatible with the Windows target. this causes the underlines in other cases. We were able to avoid these problems by implementing a custom IntelliSense provider in Visual Studio Code, but on VS 2022 this is a much more complex task. For now the recommendation is to include the Windows project there even if it is not going to be used as a production environment - it is useful for IntelliSense and even debugging.

**Comment by MartinZikmund** [comment]
It would be useful if we could automatically detect that non-Windows target is selected when a `.xaml` file is opened and switch it for the user, but it does not resolve the situation when the Windows target is not selected in the Wizard

**Comment by aliajboy** [comment]
I have same Issue on .Net 7 and Nothing Helps. 
Here, See for your selves:
![Untitled](https://github.com/unoplatform/uno/assets/73298440/812613ae-507c-447c-8f90-6f9c37ca65c0)
